snippets/core/system-text-json/csharp/WeatherForecastRuntimeIgnoreConverter.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace SystemTextJsonSamples
+{
+    public class WeatherForecastRuntimeIgnoreConverter : JsonConverter<WeatherForecast>
+    {
+        public override WeatherForecast Read(
+            ref Utf8JsonReader reader, 
+            Type typeToConvert, 
+            JsonSerializerOptions options)
+        {
+            if (reader.TokenType != JsonTokenType.StartObject)
+            {
+                throw new JsonException();
+            }
+
+            var wf = new WeatherForecast();
+
+            while (reader.Read())
+            {
+                if (reader.TokenType == JsonTokenType.EndObject)
+                {
+                    return wf;
+                }
+
+                if (reader.TokenType == JsonTokenType.PropertyName)
+                {
+                    var propertyName = reader.GetString();
```

Comment:
nit: don't use var here.
```suggestion
                    string propertyName = reader.GetString();
```